ios/FluentUI/Vnext/ActivityIndicator/ActivityIndicatorTokens.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Pre-defined sizes of the activity indicator
+@objc public enum MSFActivityIndicatorSize: Int, CaseIterable {
+    case xSmall
+    case small
+    case medium
+    case large
+    case xLarge
+
+    var size: CGFloat {
+        switch self {
+        case .xSmall:
+            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.xSmall
+        case .small:
+            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.small
+        case .medium:
+            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.medium
+        case .large:
+            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.large
+        case .xLarge:
+            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.xLarge
+        }
+    }
+}
+
+/// Representation of design tokens to buttons at runtime which interfaces with the Design Token System auto-generated code.
+/// Updating these properties causes the SwiftUI button to update its view automatically.
+class MSFActivityIndicatorTokens: MSFTokensBase, ObservableObject {
+    @Published public var activityIndicatorSize: CGFloat!
+    @Published public var thickness: CGFloat!
+    @Published public var textFont: UIFont!
+
+    @Published public var defaultColor: UIColor!
```

Comment:
I'd be interested to know if these tokens are the same sort we'd want to support for an eventual react native counterpart

Reply:
This is a non-official list of tokens of the very basic configuration we're likely going to provide.
I think it's a good list to get the conversation started, but we definitely need to talk with design, PM and other platforms/library (like FURN :) ) to get to a coherent list across the board.